README.md

```diff
@@ -274,6 +274,7 @@ Docker EE is on the same code base as Docker CE, so also built from Moby, with c
 
 #### Desktop
 
+* [Captain](https://getcaptain.co/) <img src="https://www.apple.com/favicon.ico" width="20" height="20"/> - Manage containers from the menu bar by [@RickWong](https://github.com/rickwong)
```

Comment:
@Moshe-Immerman What is that image in the middle of the description?

Reply:
This is mac only -  I can replace it with text or a different/hosted image?

Reply:
It's better if you just write "For Mac OS" in the description.